apps/gitness/src/i18n/i18n.ts

```diff
@@ -0,0 +1,57 @@
+import { initReactI18next } from 'react-i18next'
+
+import i18n from 'i18next'
+import LanguageDetector from 'i18next-browser-languagedetector'
+
+const languageDetectorOptions = {
+  // Order and from where user language should be detected
+  order: ['localStorage', 'navigator', 'cookie'],
+
+  // Keys to search language in
+  lookupCookie: 'i18next',
+  lookupLocalStorage: 'i18nextLng',
+
+  // Cache user language on
+  caches: ['cookie', 'localStorage']
+}
+
+i18n
+  .use(initReactI18next)
+  .use(LanguageDetector)
+  .init({
+    detection: languageDetectorOptions,
+    resources: {},
+    fallbackLng: 'en',
+    // debug: true,
+    react: {
+      bindI18n: 'languageChanged',
+      bindI18nStore: 'added'
+    },
+    interpolation: {
+      escapeValue: false
+    }
+  })
+
+export const handleLanguageChange = (lng: string) => {
+  // set system language
+  if (lng === 'system') {
+    const navigatorLang = navigator.language.split('-')[0]
```

Comment:
Do we need to use `navigator.userLanguage` as a fallback here? As will IE return user selection as `userLanguage`.  Can we use something like this? 
 
```ts
(navigator.language || navigator.userLanguage || 'en').split('-')[0]
```